src/platformSpecific.android.js

```diff
@@ -66,7 +66,7 @@ function navigatorSetButtons(navigator, navigatorEventID, params) {
 }
 
 function navigatorPop(navigator, params) {
-  RctActivity.navigatorPop(navigator);
+  RctActivity.navigatorPop(utils.stripRefs(navigator));
```

Comment:
Buttons that were bound on screen load seemed to work ok, but the dynamic buttons on the second screen were throwing a circular reference error.